src/map/homunculus.cpp

```diff
@@ -807,12 +782,19 @@ int hom_decrease_intimacy(struct homun_data * hd, unsigned int value)
 }
 
 /**
-* Update homunculus info to master after healing
-* @param hd
+* Update homunculus status after healing/damage
+* @param hd Homunculus data
+* @param hp HP amount
+* @param sp SP amount
 */
-void hom_heal(struct homun_data *hd) {
-	if (hd->master)
-		clif_hominfo(hd->master,hd,0);
+void hom_heal(struct homun_data *hd, int hp, int sp) {
```

Comment:
If you already touch the function definition, please change hd to reference and hp/sp to int32.

Reply:
Right!